src/model/input.ts

```diff
@@ -20,6 +21,7 @@ export class Input {
     this.prSource = core.getInput('pr_source', {required: true})
     this.prTarget = core.getInput('pr_target')
     this.prBody = core.getInput('pr_body')
+    this.prBodyWithLinks = core.getInput('pr_body_with_links') === 'true' // getBooleanInput() raises TypeError!
```

Comment:
might have been caused by the old `@actions/core` version, if you don't feel like retesting that, then I can do that later too :slightly_smiling_face: